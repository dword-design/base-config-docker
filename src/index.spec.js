import execa from 'execa'
import outputFiles from 'output-files'
import P from 'path'
import withLocalTmpDir from 'with-local-tmp-dir'

export default {
  works() {
    return withLocalTmpDir(async () => {
      await outputFiles({
        'node_modules/base-config-self/index.js':
          "module.exports = require('../../../src')",
        'package.json': JSON.stringify({
          baseConfig: 'self',
          name: '@dword-design/docker-foo',
        }),
      })
      await execa.command('base prepare')
      expect(require(P.resolve('.releaserc.json'))).toMatchSnapshot(this)
    })
  },
}
