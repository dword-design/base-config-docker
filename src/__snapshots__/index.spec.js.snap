// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index works 1`] = `
Object {
  "plugins": Array [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    "@semantic-release/changelog",
    Array [
      "@semantic-release/npm",
      Object {
        "npmPublish": false,
      },
    ],
    Array [
      "semantic-release-docker",
      Object {
        "name": "dworddesign/foo",
      },
    ],
    "@semantic-release/github",
    Array [
      "@semantic-release/git",
      Object {
        "message": "chore: \${nextRelease.version} [skip ci]

\${nextRelease.notes}",
      },
    ],
  ],
}
`;
